qemu/tests/win_virtio_driver_installer_uninstall.py

```diff
@@ -70,18 +71,21 @@ def run(test, params, env):
                           test.log.info)
     vm.send_key('meta_l-d')
     time.sleep(30)
-    run_uninstall_cmd = utils_misc.set_winutils_letter(session,
-                                                       params["run_uninstall_cmd"])
-
-    session = run_installer_with_interaction(vm, session, test, params,
-                                             run_uninstall_cmd)
 
     if uninstall_method == "msi":
+        run_uninstall_msi_cmd = utils_misc.set_winutils_letter(
+            session, params["run_uninstall_msi_cmd"]
+        )
+        session.cmd(run_uninstall_msi_cmd)
```

Comment:
From my point of view, this change makes the code redundant and the function does not change at all, I also don't see the point why we have to change the name of `run_uninstall_cmd`.

Reply:
Only have
```
session.cmd(run_uninstall_cmd)
time.sleep(30)
```
Should it work as expected?

Reply:
Just checked the previous discussion, I understand that another name might seem clearer for you, but `by_msi` as the test case name would be sufficient from my point of view, and another name means locate extra memory in system, which is less efficient.

Reply:
> From my point of view, this change makes the code redundant and the function does not change at all, I also don't see the point why we have to change the name of `run_uninstall_cmd`.

Let me explain more, run_uninstall_msi_cmd couldn't be run in  run_installer_with_interaction() function, in the initial design run_uninstall_msi_cmd doesn't complete the whole uninstallation, it just capture some warning info, while run_installer_with_interaction() function has some steps to wait system rebooing at the end of installation.

Reply:
> Just checked the previous discussion, I understand that another name might seem clearer for you, but `by_msi` as the test case name would be sufficient from my point of view, and another name means locate extra memory in system, which is less efficient.

I changed the name because they are different ones, and  they are not used in the same function like before.
   ` session = run_installer_with_interaction(vm, session, test, params,run_uninstall_cmd)`